packaging/nntrainer.spec

```diff
@@ -321,44 +321,18 @@ meson --buildtype=plain --prefix=%{_prefix} --sysconfdir=%{_sysconfdir} \
 ninja -C build %{?_smp_mflags}
 
 %if 0%{?unit_test}
-# independent unittests of nntrainer
-# @todo: we only need single line of this.
-%define meson_test meson test -C build -t 2.0 --print-errorlogs
-%{meson_test} unittest_tizen_capi
-%{meson_test} unittest_tizen_capi_layer
-%{meson_test} unittest_tizen_capi_optimizer
-%{meson_test} unittest_tizen_capi_dataset
-%{meson_test} unittest_nntrainer_activations
-%{meson_test} unittest_nntrainer_internal
-%{meson_test} unittest_nntrainer_layers
-%{meson_test} unittest_nntrainer_lazy_tensor
-%{meson_test} unittest_nntrainer_tensor
-%{meson_test} unittest_util_func
-%{meson_test} unittest_databuffer_file
-%{meson_test} unittest_nntrainer_modelfile
-%{meson_test} unittest_nntrainer_models
-%{meson_test} unittest_nntrainer_graph
-%{meson_test} unittest_nntrainer_appcontext
-%{meson_test} unittest_nntrainer_profiler
-%{meson_test} unittest_ccapi
-%{meson_test} app_knn
-%{meson_test} app_logistic
-%{meson_test} app_mnist
-%{meson_test} app_cifar_classification
-%{meson_test} app_cifar_classification_func
-# %{meson_test} app_draw_classification
-%{meson_test} app_plugin_test
-%{meson_test} simpleshot_tests
-
+export NNSTREAMER_CONF=$(pwd)/test/nnstreamer/nnstreamer-test.ini
```

Comment:
I wonder if this is only required for ssat test, then this can be moved to inside of `%if 0%{?nnstreamer_filter}` as meson test sets their environment variables when defining the tests.

Reply:
DrawClassification also requires this for now. 

There is a minor issue with including conf (which is only needed for the tizen build) so it is left for now. 
Later, `nnstremaer-test.ini` will be deprecated in favor of `nnstreamer-test.ini.in`, at the same time, this will migrate into meson test.

Reply:
sounds good.